docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,185 @@
 # Arcaflow Getting Started Guide
 
-## Step 1: Get a container engine
+## Running Workflows
+An Arcaflow workflow is a definition of steps structured together to perform complex actions. Workflows are defined as machine-readable YAML and therefore can be version-controlled and shared easily to run in different environments. A workflow is a way of encapsulating and sharing expertise and ensuring reproducible results.
 
-In order to use Arcaflow, you will need a container engine on your computer. For the purposes of this guide, we'll assume you are using Docker or Podman.
+Running a workflow only requires having the [Arcaflow engine binary](https://github.com/arcalot/arcaflow-engine/releases), the workflow definition file, and in most cases an input file. An optional config file also allows for setting workflow defaults, such as log levels. All of these files are typically defined in YAML. Additionally, the target of the workflow needs a compatible container platform, such as Podman, Docker, or Kubernetes.
 
-## Step 2: Get the engine
+!!! note
+    The default container platform for the Arcaflow engine is Podman. To use another platform, a [configuration file](/arcaflow/running/setup/#configuration) is required.
 
-Head on over to the [GitHub releases page](https://github.com/arcalot/arcaflow-engine/releases) and download the latest release.
+A repository of [example workflows](https://github.com/arcalot/arcaflow-workflows) is available for reference and practice. Let's try running the [basic example](https://github.com/arcalot/arcaflow-workflows/basic-examples/basic).
 
-## Step 3: Create your first plugin
-
-Let's create a simple hello-world plugin in Python. We'll publish the code here, you can find the details in the [Python plugin guide](plugins/python/first.md).
-
-```python title="plugin.py"
-#!/usr/local/bin/python3
-import dataclasses
-import sys
-from arcaflow_plugin_sdk import plugin
+First we will clone the example workflows repository:
 
+```bash
+git clone https://github.com/arcalot/arcaflow-workflows.git
+```
 
-@dataclasses.dataclass
-class InputParams:
-    name: str
-    
-    
-@dataclasses.dataclass
-class SuccessOutput:
-    message: str
+Then we will run the workflow, setting the workflow directory as the context, and defining the workflow, configuration, and input files to use:
 
+```bash
+arcaflow --context arcaflow-workflows/basic-examples/basic/ \
+--workflow workflow.yaml --config config.yaml --input input.yaml
+```
```

Comment:
Because this is potentially the user's very first encounter with the command, I prefer to leave this example as verbose.  The very next example in the page of running a workflow includes a Tip noting that the `--workflow` parameter is not required. A getting started guide needs to build knowledge piece-by-piece.